FirebaseRemoteConfigSwift/Sources/PropertyWrapper/RemoteConfigValueObservable.swift

```diff
@@ -61,13 +55,19 @@ internal class RemoteConfigValueObservable<T: Decodable>: ObservableObject {
     if FirebaseApp.app()?.name != appName {
       return
     }
+    applyConfigValue()
+  }
+
+  private func applyConfigValue() {
     do {
       let configValue: RemoteConfigValue = remoteConfig[key]
-      if configValue.source == .remote {
+      if configValue.source == .remote || configValue.source == .default {
         self.configValue = try remoteConfig[key].decoded()
+      } else {
+        self.configValue = fallbackValue
       }
     } catch {
-      // Suppresses a hard failure if decoding failed.
```

Comment:
I don't think this is right, we should only get remote value since this method is only triggered when config has remote value needs to be activated.